docs/enhancements/soft-install-timeout.md

```diff
@@ -0,0 +1,122 @@
+---
+title: soft-install-timeout
+authors:
+- "@jhernand"
+creation-date: 2023-10-04
+last-updated: 2022-10-04
+---
+
+# Soft install timeout
+
+## Summary
+
+Currently we have an installation timeout of one hour. When that expires the
+cluster is automatically moved to the error state. We want to change that to a
+warning, so that users are informed that the installation is taking longer than
+expected and the installation will continue.
+
+## Motivation
+
+This is important because users may be able to fix he issues that delay the
+installation instead of having to start it over.
+
+### Goals
+
+### Non-Goals
+
+It is not a goal to change the global installation timeout configured via the
+`INSTALLATION_TIMEOUT` environment variable. That is set by default to `24h`
+and at that point the assisted service stops monitoring the cluster.
```

Comment:
What do we gain by stopping monitoring of a cluster whose installation hasn't completed? Is it much of a resource burden for us?

Reply:
> After 24 hours the installation will fail anyway due to expired certificates, so there isn't much value in continuing.

Unless the cluster is up and running, but one cluster operator is degraded?

Reply:
True.